tools/deb_to_rb.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+
+set -e
+
+CREW_PREFIX=/usr/local
+CREW_LIB_PREFIX=${CREW_PREFIX}/lib
+pkg_prefix=${CREW_LIB_PREFIX}/crew/packages
+GREEN='\e[1;32m';
+RED='\e[1;31m';
+BLUE='\e[1;34m';
+YELLOW='\e[1;33m';
+RESET='\e[0m'
+
+rm -rf ${out}
```

Comment:
Where is `${out}` set earlier in the script?

Reply:
Removed